pkg/crd/init.go

```diff
@@ -155,14 +173,18 @@ func readCustomColumns(t reflect.Type, path string) (result []apiextv1.CustomRes
 		if t.Kind() == reflect.Ptr {
 			t = t.Elem()
 		}
-		if t.Kind() == reflect.Struct {
+		if k := openAPISchemaKind(t); k != reflect.Invalid {
+			if col, ok := tagToColumn(f, k, path+fieldName); ok {
+				result = append(result, col)
+			}
+		} else if t.Kind() == reflect.Struct {
 			if f.Anonymous {
 				result = append(result, readCustomColumns(t, path)...)
 			} else {
-				result = append(result, readCustomColumns(t, path+"."+fieldName)...)
+				result = append(result, readCustomColumns(t, path+fieldName+".")...)
```

Comment:
Was this just incorrect before? I the topmost call would already pass "." as the path.

Reply:
Yes, this was wrong before. The root path is . so it would generate things like ..spec instead of .spec. as the parent path for nested fields.

Reply:
Yeah, I also noticed that the path variable just wasn't used before 😆